scripts/rebase.sh

```
@@ -427,7 +443,7 @@ rebase_to() {
         title "## Updating vendor directory"
         go mod vendor
         title "## Patching vendor directory"
-        git apply scripts/rebase_patches/*
+        git apply scripts/rebase_patches/*.patch 2>/dev/null || true
```

Why hide the errors from this step?

---

Right, I'd keep those,  otherwise it can be hard to trace

---

Soon having patches to apply will be the exception and I was bothered by CI highlighting the expected. But I agree and will instead apply conditionally on whether patches exist.

---

done